src/main/kotlin/com/cognifide/gradle/aem/tooling/tail/Tailer.kt

```diff
@@ -0,0 +1,44 @@
+package com.cognifide.gradle.aem.tooling.tail
+
+import java.io.BufferedReader
+
+class Tailer(private val source: LogSource, private val destination: LogDestination, val name: String = "") {
+
+    private val parser = Parser()
+
+    private var lastLogChecksum = ""
+
+    fun tail() {
+        val logs = source.processChunk { chunkReader ->
```

Comment:
replace with `it`. 

general rule, use `it` when expr is single line; more lines then introduce name (this is a rule from detekt)

Reply:
OK